client/blocks/login/style.scss

```diff
@@ -236,6 +236,7 @@ $breakpoint-mobile: 782px; //Mobile size.
 .login__form-action {
 	button {
 		width: 100%;
+		height: 44px;
```

Comment:
Login form inputs and submit buttons are taller across the board (44px vs 40px). This might be temporary if we decide to standardize them.